views/game.erb

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Mages vs. Templars</title>
+	<link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+
+  <h2>Mages vs. Templars</h2>
+
+  <h3 id = "message"></h3>
+
+	<div class = "container">
+		<table>
```

Comment:
For funsies/extra challenge, check out flex box/grid layouts.

Flexbox:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Using_CSS_flexible_boxes
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

CSS grid:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout
https://css-tricks.com/snippets/css/complete-guide-grid/